website/docs/reference/node-selection/test-selection-examples.md

```diff
@@ -62,30 +62,55 @@ The "buildable" and "cautious" modes can be useful in environments when you're o
 
-These are the modes to configure the behavior when performing indirect selection (with `eager` as the default):
+You can use the following modes to configure the behavior when performing indirect selection (with `eager` as the default). Test exclusion is always greedy: if ANY parent is explicitly excluded, the test will be excluded as well.
 
-1. `eager` (default) - include ANY test that references the selected nodes
-1. `cautious` - restrict to tests that ONLY refer to selected nodes
-1. `buildable` -  restrict to tests that ONLY refer to selected nodes (or their ancestors)
-1. `empty` - restrict to tests that are only for the selected node and ignore all tests from the attached nodes 
+The `buildable`, `cautious`, and `empty` modes can be useful in environments when you're only building a subset of your DAG, and you want to avoid test failures in `eager` mode caused by unbuilt resources. You can also achieve this with [deferral](/reference/node-selection/defer).
 
-Note that test exclusion is always greedy: if ANY parent is explicitly excluded, the test will be excluded as well.
+#### Eager mode (default)
+
+  By default, runs tests if any of their parent nodes are selected, regardless of whether all dependencies are met. This includes ANY tests that reference the selected nodes. Models will be built if they depend on the selected model. In this mode, any tests depending on unbuilt resources will raise an error, helping to identify potential issues.
+
+#### Cautious mode
+
+  Ensures that tests are executed and models are built only when all necessary dependencies of the selected models are met. Restricts tests to only those that exclusively reference selected nodes. Tests will only be executed if all the nodes they depend on are selected, which prevents tests from running if one or more of its parents nodes are unselected and, consequently, unbuilt.
+
+#### Buildable mode
+
+  Only runs tests that refer to selected nodes (or their ancestors). Only builds models that rely on the selected nodes (or their ancestors). This mode is slightly more inclusive than "cautious" by including tests whose references are each within the selected nodes (or their ancestors). This mode is useful when a test depends on a model **and** a direct ancestor of that model, like confirming an aggregation has the same totals as its input.
+
+#### Empty mode
+
+  Restrict to tests that are only for the selected node and ignore all tests from the attached nodes 
 
-The "buildable", "cautious", and "empty" modes can be useful in environments when you're only building a subset of your DAG, and you want to avoid test failures in "eager" mode caused by unbuilt resources. (Another way to achieve this is with [deferral](/reference/node-selection/defer)).
+To visualize these methods, suppose you have `model_a`, `model_b`, and `model_c` and associated data tests. Here's what it will look like when you execute `dbt run` and then the various indirect selction modes:
```

Comment:
```suggestion
To visualize these methods, suppose you have `model_a`, `model_b`, and `model_c` and associated data tests. Here's what it will look like when you execute `dbt run` vs. `dbt build` with the various indirect selection modes:
```